packages/subgraph/schema.graphql

```diff
@@ -203,8 +203,8 @@ type TokenVotingPlugin implements IPlugin @entity {
   id: ID!
   daos: [DaoPlugin!] @derivedFrom(field: "plugin")
   proposals: [TokenVotingProposal!]! @derivedFrom(field: "plugin")
-  relativeSupportThresholdPct: BigInt
-  totalSupportThresholdPct: BigInt
+  supportThreshold: BigInt
+  participationThreshold: BigInt
```

Comment:
Reminder about the ordering of parameters

Reply:
Fixed globally in 594260c92b86f87970a9060d9e8700272d77cfdd